prisma/models/.config.prisma

```diff
@@ -5,4 +5,4 @@ generator client {
 datasource db {
   provider = "postgresql"
   url      = env("DATABASE_URL")
-}
\ No newline at end of file
+}
```

Comment:
maybe remove unnecessary changes

Reply:
i have no record of this commit showing up on my branch, and i don't recall touching .config.prisma recently.
i'll try to avoid useless diffs like this in the future nonetheless.